src/app/tabs/newAdmin/textrecords/components/DisplayTextRecordComponent.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { multilanguage } from 'redux-multilanguage';
+import EditDeleteTextRecordComponent from './EditDeleteTextRecordComponent';
+
+const DisplayTextRecordComponent = ({
+  value,
+}) => {
+  const filteredValues = value.filter(c => c.result !== '');
+  return (
+    <div>
+      <table className="table table-bordered">
```

Comment:
The data that is being listed is not tabular data so using a table isn't appropriate. Instead we should think about using Bootstrap's grid (rows and columns) instead. There are two reasons for this, 1. it will be better on mobile when it collapses down and the table will not go off the screen. 2. it is better for accessibility, specifically users with low or no vision where they use a screen reader to read the content back to them. 

Here is what it looks like on mobile for example:

![Screen Shot 2022-07-06 at 3 14 22 PM](https://user-images.githubusercontent.com/766679/177547698-1650cc9f-a80b-4f98-8683-817de9749e02.png)

Reply:
To match consistency with the other parts of the site, you could use the existing `AddressInputComponent`. Although it is named "address" it is used for subdomains, reverse, coin addresses. This would also fix the table issue.

![Screen Shot 2022-07-06 at 3 18 28 PM](https://user-images.githubusercontent.com/766679/177548430-e038bb7a-2668-493e-81c4-80c2730be2f1.png)